### DISCUSSION TITLE
Hallo

### DISCUSSION THREAD

**Discussion by BigBerf** [makerforgetech/modular-biped#68] [General]
Hallo Leute , das Roboterprojekt ist echt faszinierend und ich möchte es sehr gerne ausprobieren. Der Drucker läuft schon auf Hochtouren. Ich hoffe ich bekomme es auch so hin wie Ihr.

**Comment by WhoIsMrSentry** [comment]
Hallo @BigBerf, willkommen in der Community. Ich habe keinen Zweifel daran, dass auch Sie den Roboter bauen werden. Vertraue einfach dir selbst.😉

**Comment by danic85** [comment]
Hi! Great to hear - let us know if you have any issues with the files. The v4 head (from the latest videos) is [here](https://cad.onshape.com/documents/2ff2b42ab754d3580a946425/w/3232eb421d270cc65bbd5055/e/6669b5a071858383fde77b3c), but it is still a work-in-progress, so expect that there will be changes.

**Comment by BigBerf** [comment]
![P_20240108_181525.jpg](https://github.com/makerforgetech/modular-biped/assets/155828427/7f473651-0801-4f76-b4d0-8d2f62296dfc)

**Reply by BigBerf** [reply]
So jetzt wieder auf Teile warten 🙂. Ich habe jetzt schon Panik wegen dem programmieren 😰

**Reply by WhoIsMrSentry** [reply]
Es besteht kein Grund zur Panik, wenn es ein Problem gibt, sind wir nur eine Gemeinde entfernt.😉

**Reply by danic85** [reply]
Looks great! 

The software is a bit of work depending on what you want to do, but it's just configuration and installing dependencies. Not too bad :+1:

**Comment by BigBerf** [comment]
![P_20240204_144618.jpg](https://github.com/makerforgetech/modular-biped/assets/155828427/7ef9e4d7-afbb-47dd-9c31-b055919aa41a)

Hallo , es ging wieder ein wenig voran.

**Comment by BigBerf** [comment]
![P_20240204_144604.jpg](https://github.com/makerforgetech/modular-biped/assets/155828427/2e59e8a4-501f-4973-9cf4-1ed2121cf034)

**Comment by danic85** [comment]
Very nice! I think you'll find my next video useful 😂
